etc/workflows/studio-upload.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>studio-upload</id>
+  <title>Studio Upload</title>
+  <tags>
+    <tag>archive</tag>
+    <tag>upload</tag>
+  </tags>
```

Comment:
If it's for studio, then it should not have these tags

```suggestion
```

Reply:
> If it's for studio, then it should not have these tags

These tags are important for the case studio files should be ingested manually using download option of the video files in studio.

Reply:
I think it's confusing to have this additional workflow but we could remove the tags and keep the configuration panel. That would make it easy for users to make it visible if they want to.

Reply:
I was outvoted on this. Let's keep this as it is